src/downzip-sw.js

```diff
@@ -83,24 +83,33 @@ self.addEventListener('fetch', async (event) => {
             // Append all the downloaded data
             try {
                 await new Promise((resolve, reject) => {
-                    fetch(file.downloadUrl).then(response => response.body).then(async (stream) => {
-                        const reader = stream.getReader()
-                        let doneReading = false
-                        while (!doneReading) {
-                            const chunk = await reader.read()
-                            const {done, value} = chunk
-
-                            if (done) {
-                                // If this stream has finished, resolve and return
-                                resolve()
-                                doneReading = true
-                            } else {
-                                // If not, append data to the zip
-                                zipMap[id].zip.appendData(value)
+                    fetch(file.downloadUrl).then(async response => {

```

Comment:
@keith-waters it seems like we should add an extra `catch` to the promise returned by the `fetch` call, probably should just do `reject(err)` there as previously